src/sx1261_2/mod.rs

```diff
@@ -320,10 +323,14 @@ where
         Ok(())
     }
 
-    // Set the power regulators operating mode to DC_DC.  Using only LDO implies that the Rx/Tx current is doubled.
     async fn set_regulator_mode(&mut self) -> Result<(), RadioError> {
-        let op_code_and_regulator_mode = [OpCode::SetRegulatorMode.value(), RegulatorMode::UseDCDC.value()];
-        self.intf.write(&op_code_and_regulator_mode, false).await
+        // SX1261/2 can use optional DC-DC to reduce power usage,
+        // but this is related to the hardware implementation of the board.
+        if self.config.use_dcdc {
+            let reg_data = [OpCode::SetRegulatorMode.value(), RegulatorMode::UseDCDC.value()];
```

Comment:
What is the default if regulator mode is not set? Also can we trust the default? Should here be an else clause?

Reply:
Default is to use LDO